diff --git a/lib/Linux/Ext2/Attributes.pm b/lib/Linux/Ext2/Attributes.pm
index e9307f8..1456e3c 100755
--- a/lib/Linux/Ext2/Attributes.pm
+++ b/lib/Linux/Ext2/Attributes.pm
@@ -58,7 +58,7 @@ our @EXPORT_OK = qw(
 	EXT2_TOPDIR_FL
 	EXT2_NOTAIL_FL
 	EXT2_UNRM_FL
-	EXT2_NOCOMP_FL
+	EXT2_NOCOMPR_FL
 	EXT2_DIRTY_FL
 );
 
@@ -81,7 +81,7 @@ our %EXPORT_TAGS = (
 		EXT2_TOPDIR_FL
 		EXT2_NOTAIL_FL
 		EXT2_UNRM_FL
-		EXT2_NOCOMP_FL
+		EXT2_NOCOMPR_FL
 		EXT2_DIRTY_FL
 	) ]
 );
@@ -107,7 +107,7 @@ use constant EXT2_SYNC_FL         => Linux::Ext2::Attributes::Constants::EXT2_SY
 use constant EXT2_TOPDIR_FL       => Linux::Ext2::Attributes::Constants::EXT2_TOPDIR_FL;
 use constant EXT2_NOTAIL_FL       => Linux::Ext2::Attributes::Constants::EXT2_NOTAIL_FL;
 use constant EXT2_UNRM_FL         => Linux::Ext2::Attributes::Constants::EXT2_UNRM_FL;
-use constant EXT2_NOCOMP_FL       => Linux::Ext2::Attributes::Constants::EXT2_NOCOMP_FL;
+use constant EXT2_NOCOMPR_FL      => Linux::Ext2::Attributes::Constants::EXT2_NOCOMPR_FL;
 use constant EXT2_DIRTY_FL        => Linux::Ext2::Attributes::Constants::EXT2_DIRTY_FL;
 
 # Constants pilfered out of the kernel headers as they don't seem to be commonly
@@ -119,7 +119,7 @@ use constant EXT4_HUGE_FILE_FL    => 0x00040000;
 
 # These flags may not be set or cleared by the user.
 
-use constant EXT2_READONLY_FLAGS  => (EXT2_ECOMPR_FL | EXT4_EXTENTS_FL | EXT4_HUGE_FILE_FL | EXT2_INDEX_FL | EXT2_NOCOMP_FL | EXT2_DIRTY_FL);
+use constant EXT2_READONLY_FLAGS  => (EXT2_ECOMPR_FL | EXT4_EXTENTS_FL | EXT4_HUGE_FILE_FL | EXT2_INDEX_FL | EXT2_NOCOMPR_FL | EXT2_DIRTY_FL);
 
 my %flag_chars = (
 	'A' => EXT2_NOATIME_FL,
@@ -139,7 +139,7 @@ my %flag_chars = (
 	'T' => EXT2_TOPDIR_FL,
 	't' => EXT2_NOTAIL_FL,
 	'u' => EXT2_UNRM_FL,
-	'X' => EXT2_NOCOMP_FL,
+	'X' => EXT2_NOCOMPR_FL,
 	'Z' => EXT2_DIRTY_FL,
 );
 
@@ -494,7 +494,7 @@ in the Linux kernel. For more information see the chattr man page.
   EXT2_TOPDIR_FL       (T) - Directory is at the top of a hierarchy.
   EXT2_NOTAIL_FL       (t) - Disable tail merging.
   EXT2_UNRM_FL         (u) - Keep file for undeletion.
-  EXT2_NOCOMP_FL       (X) - Don't compress file.
+  EXT2_NOCOMPR_FL      (X) - Don't compress file.
   EXT2_DIRTY_FL        (Z) - Compressed file is dirty.
 
 =head1 COPYRIGHT AND LICENSE
diff --git a/lib/Linux/Ext2/Attributes/Constants.pm.PL b/lib/Linux/Ext2/Attributes/Constants.pm.PL
index 36e5e9b..5f4811e 100644
--- a/lib/Linux/Ext2/Attributes/Constants.pm.PL
+++ b/lib/Linux/Ext2/Attributes/Constants.pm.PL
@@ -3,7 +3,7 @@ use ExtUtils::H2PM;
 module "Linux::Ext2::Attributes::Constants";
 
 include "linux/fs.h";
-include "linux/ext2_fs.h";
+include "ext2fs/ext2_fs.h";
 
 constant "EXT2_IOC_GETFLAGS";
 constant "EXT2_IOC_SETFLAGS";
@@ -22,7 +22,7 @@ constant "EXT2_SYNC_FL";
 constant "EXT2_TOPDIR_FL";
 constant "EXT2_NOTAIL_FL";
 constant "EXT2_UNRM_FL";
-constant "EXT2_NOCOMP_FL";
+constant "EXT2_NOCOMPR_FL";
 constant "EXT2_DIRTY_FL";
 
 write_output $ARGV[0];
diff --git a/t/11-attributes.t b/t/11-attributes.t
index c03122b..6a0f30b 100644
--- a/t/11-attributes.t
+++ b/t/11-attributes.t
@@ -112,4 +112,4 @@ sub test_strip {
 
 test_strip(EXT2_NOATIME_FL, EXT2_NOATIME_FL);
 test_strip(EXT2_NOATIME_FL | EXT4_EXTENTS_FL, EXT2_NOATIME_FL);
-test_strip(EXT4_EXTENTS_FL | EXT2_NOCOMP_FL, 0);
+test_strip(EXT4_EXTENTS_FL | EXT2_NOCOMPR_FL, 0);
