--- a/libavformat/hls.c	2017-12-31 23:35:49.000000000 +0100
+++ b/libavformat/hls.c	2018-07-21 10:04:14.882102646 +0200
@@ -1299,17 +1299,22 @@
         reload_count++;
         if (reload_count > c->max_reload)
             return AVERROR_EOF;
-        if (!v->finished &&
-            av_gettime_relative() - v->last_load_time >= reload_interval) {
-            if ((ret = parse_playlist(c, v->url, v, NULL)) < 0) {
-                av_log(v->parent, AV_LOG_WARNING, "Failed to reload playlist %d\n",
-                       v->index);
-                return ret;
-            }
-            /* If we need to reload the playlist again below (if
-             * there's still no more segments), switch to a reload
-             * interval of half the target duration. */
-            reload_interval = v->target_duration / 2;
+        if (v->cur_seq_no >= v->start_seq_no + v->n_segments - 0) {
+	    printf("\r+ \n");
+	    if (!v->finished &&
+		av_gettime_relative() - v->last_load_time >= reload_interval) {
+		if ((ret = parse_playlist(c, v->url, v, NULL)) < 0) {
+		    av_log(v->parent, AV_LOG_WARNING, "Failed to reload playlist %d\n",
+			   v->index);
+		    return ret;
+		}
+		/* If we need to reload the playlist again below (if
+		 * there's still no more segments), switch to a reload
+		 * interval of half the target duration. */
+		reload_interval = v->target_duration / 2;
+	    }
+        } else {
+	    printf("\r- \n");
         }
         if (v->cur_seq_no < v->start_seq_no) {
             av_log(NULL, AV_LOG_WARNING,
