--- a/gajim/common/app.py	2018-11-06 23:57:20.088079001 +0100
+++ a/gajim/common/app.py	2018-11-06 23:58:36.638834033 +0100
@@ -299,17 +299,7 @@
         pass
 
     # GSPELL
-    try:
-        gi.require_version('Gspell', '1')
-        from gi.repository import Gspell
-        langs = Gspell.language_get_available()
-        for lang in langs:
-            log('gajim').info('%s (%s) dict available',
-                              lang.get_name(), lang.get_code())
-        if langs:
-            _dependencies['GSPELL'] = True
-    except (ImportError, ValueError):
-        pass
+    # [REMOVED BY EBUILD]
 
     # Print results
     for dep, val in _dependencies.items():
